src/main/java/org/jabref/gui/libraryproperties/general/GeneralPropertiesView.java

```diff
@@ -59,6 +65,14 @@ public void initialize() {
         userSpecificFileDirectory.textProperty().bindBidirectional(viewModel.userSpecificFileDirectoryProperty());
         laTexFileDirectory.textProperty().bindBidirectional(viewModel.laTexFileDirectoryProperty());
 
+        librarySpecificFileDirectoryValidationVisualizer.setDecoration(new IconValidationDecorator());
+        userSpecificFileDirectoryValidationVisualizer.setDecoration(new IconValidationDecorator());
+        laTexFileDirectoryValidationVisualizer.setDecoration(new IconValidationDecorator());
+
+        Platform.runLater(() -> librarySpecificFileDirectoryValidationVisualizer.initVisualization(viewModel.librarySpecificFileDirectoryStatus(), librarySpecificFileDirectory));
```

Comment:
Just one Platform.runLater is enough. e..g as in sharedDatabaseLogin 
<img width="850" alt="grafik" src="https://github.com/user-attachments/assets/9b34b19c-9545-42d0-a228-1d58805c9c1a" />